src/main/java/com/agata/jeeshop/controllers/RestOrderController.java

```diff
@@ -0,0 +1,25 @@
+package com.agata.jeeshop.controllers;
+
+import com.agata.jeeshop.dto.CartDto;
+import com.agata.jeeshop.models.OrderedCart;
+import com.agata.jeeshop.services.CartService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/rest/order")
+public class RestOrderController {
+    @Autowired
+    private CartService cartService;
+
+    @PostMapping
+    public ResponseEntity<OrderedCart> submitOrder(@RequestBody OrderedCart orderedCart){
+        CartDto cartDto = cartService.addToOrdered(orderedCart.getCartId());
```

Comment:
Linia 22 mówi logicznie: serwisie od koszyka, masz id koszyka, dodaj to do ordered i zwróć mi koszyk. To trochę bez sensu. Zastanów się co tak naprawdę chcesz tutaj zrobić (nie kodem, a po prostu logicznie). Według mnie scenariusz powinien być następujący. 

Dostajemy OrderedCart, który posiada cartId, następnie przekazujemy do OrderService to cartId (do metody createOrder). Serwis ten mając cartId może znaleść koszyk w bazie, oznaczyć go jako zrealizowany, utworzyć nowe zamówienie, ustawić na tym zamówieniu koszyk i zwrócić PurchaseOrderDto z podsumowaniem zamówienia.